utils/create_admin_user.py

```diff
@@ -1,54 +1,71 @@
-from flask_simpleldap import LDAP
-
-from ldap_utils.ldap_utils import refine_data
-from models.users import User, Role, RoleEnum
+from config import Config
 from init_db import db
+from ldap_utils.ldap_utils import ldap_client, refine_data
+from models.users import User, Role, RoleEnum
 
 
-ldap_client = LDAP()
+email_list = Config.ADMIN_LIST.split()
 
 
 def create_super_user():
-    email_data = input("New admin email: ")
-    user_ldap = ldap_client.get_object_details(user=email_data)
-    if user_ldap:
-        if refine_data(user_ldap, 'l') != 'Wroclaw':
-            print('Only employees from Wroclaw are accepted')
-            return
-        user_ldap_data = {
-            'mail': refine_data(user_ldap, 'mail'),
-            'givenName': refine_data(user_ldap, 'givenName'),
-            'sn': refine_data(user_ldap, 'sn'),
-            'employeeID': refine_data(user_ldap, 'employeeID')
-        }
-        user_db = User.query.filter_by(
-            employee_id=user_ldap_data['employeeID']
-        ).first()
-        if user_db:
-            user_db_data = {
-                'mail': user_db.email,
-                'givenName': user_db.first_name,
-                'sn': user_db.surname
+    global email_list
```

Comment:
Don't use global vars. It's dangerous in its nature. I think it would be the best to encapsulate this var into the function.
Or pass it as a parameter.